tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -50,9 +50,6 @@ class Backend(Enum):
 }
 
 DEFAULT_W_OB_CTR = {
-    # According to the url below, PyTorch's reference module does not support
```

Comment:
Why delete this?

Reply:
I thought this code modifications could explain the confusing question, but didn't it?